tests/acceptance/EditorHeaderBlockCest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use MailPoet\Test\DataFactories\Newsletter;
+
+require_once __DIR__ . '/../DataFactories/Newsletter.php';
+
+class EditorHeaderBlockCest {
+  function addHeader(\AcceptanceTester $I) {
+    $I->wantTo('add header block to newsletter');
+    $newsletterTitle = 'Header Block Newsletter';
+    $headerInEditor = ('[data-automation-id="header"]');
+    $headerSettingsIcon = ('[data-automation-id="settings_tool"]');
+    $headerSettingsAssertion = ('[data-automation-id="header_done_button"]');
+    (new Newsletter())
+      ->withSubject($newsletterTitle)
+      ->loadBodyFrom('newsletterWithTextNoHeader.json')
+      ->create();
+    $I->login();
+    $I->amOnMailpoetPage('Emails');
+    $I->waitForText($newsletterTitle);
+    $I->clickItemRowActionByItemName($newsletterTitle, 'Edit');
+    // Create header block
+    $I->waitForText('Header');
+    $I->wait(1); // just to be sure
```

Comment:
I really don't like the idea of having explicit waits "just to be sure". If we need it we can wait for an element to be present. If we don't need it, remove it.

Reply:
I introduced this "just to be sure" wait some time ago, so it's already in some tests. I realize it's badly commented - it's not there "just to be sure", it's there because of blocks animation when editor is loaded. All elements and texts are already visible (or in DOM), so we can't wait for that, it's just animation is moving it on the page so we need to wait for animation to finish. It's not even 1 full second, more like 0.5s. I'd ignore this in this PR, and after all this tests are merged, I'll change it everywhere so it makes more sense why that 'wait' is there.